libcontainer/cgroups/fs/fs.go

```diff
@@ -38,24 +38,11 @@ var (
 
 var errSubsystemDoesNotExist = errors.New("cgroup: subsystem does not exist")
 
-type subsystemSet []subsystem
-
-func (s subsystemSet) Get(name string) (subsystem, error) {
-	for _, ss := range s {
-		if ss.Name() == name {
-			return ss, nil
-		}
-	}
-	return nil, errSubsystemDoesNotExist
-}
-
 type subsystem interface {
 	// Name returns the name of the subsystem.
 	Name() string
 	// Returns the stats, as 'stats', corresponding to the cgroup under 'path'.
 	GetStats(path string, stats *cgroups.Stats) error
-	// Removes the cgroup represented by 'cgroupData'.
```

Comment:
This could be of use to external clients and maybe later on if/when we factor this out into a cgroups library.

Reply:
Theoretically, yes.

Practically,
1. I don't see it being used by kubernetes;
2. It uses `d.path` which is expensive, it requires to parse mountinfo (this is actually the main point of it being removed);
3. Its API is not in line with e.g. `Set` (which does not use `d.path` but the path is provided directly).

Also, if needed, it can be trivial to reimplement / re-add, as it doesn't do anything special, so I see no point in keeping it.